### PR TITLE
Trivial(qt): Capitalization, spelling

### PR DESCRIPTION
**Pull request by yahiheb**
Changed:
`block chain` to `blockchain`
`synchronisation` to `synchronization`
`bitcoin` to `Bitcoin`

### CODE REVIEW

File: src/qt/forms/intro.ui [hunk 1]
```diff
@@ -223,7 +223,7 @@
    <item>
     <widget class="QLabel" name="lblExplanation2">
      <property name="text">
-      <string>This initial synchronisation is very demanding, and may expose hardware problems with your computer that had previously gone unnoticed. Each time you run %1, it will continue downloading where it left off.</string>
```
Comment (kristapsk) [inline src/qt/forms/intro.ui]: "Synchronisation" is just British spelling of American "synchronization", don't see a need for this to be changed.

Comment (yahiheb) [reply src/qt/forms/intro.ui]: Yes, I saw in other places the American spelling was used so I thought it should be changed for consistency.

**Comment by hebasto** [comment]
@yahiheb 
Thank you.

May I suggest you two pieces of good reading:
- [How to Contribute Pull Requests to Bitcoin Core](https://jonatack.github.io/articles/how-to-contribute-pull-requests-to-bitcoin-core) by @jonatack 
- [Onboarding to Bitcoin Core](https://medium.com/@amitiu/onboarding-to-bitcoin-core-7c1a83b20365) by @amitiuttarwar

File: src/qt/bitcoingui.cpp [hunk 2]
```diff
@@ -254,7 +254,7 @@ void BitcoinGUI::createActions()
     sendCoinsMenuAction->setToolTip(sendCoinsMenuAction->statusTip());
 
     receiveCoinsAction = new QAction(platformStyle->SingleColorIcon(":/icons/receiving_addresses"), tr("&Receive"), this);
-    receiveCoinsAction->setStatusTip(tr("Request payments (generates QR codes and bitcoin: URIs)"));
+    receiveCoinsAction->setStatusTip(tr("Request payments (generates QR codes and Bitcoin: URIs)"));
```
Comment (fanquake) [inline src/qt/bitcoingui.cpp]: There are both meant to be `bitcoin:`.

**Comment by yahiheb** [comment]
> @yahiheb
> Thank you.
> 
> May I suggest you two pieces of good reading:
> 
> * [How to Contribute Pull Requests to Bitcoin Core](https://jonatack.github.io/articles/how-to-contribute-pull-requests-to-bitcoin-core) by @jonatack
> * [Onboarding to Bitcoin Core](https://medium.com/@amitiu/onboarding-to-bitcoin-core-7c1a83b20365) by @amitiuttarwar

@hebasto Thank you. I will definitely take a look at that.

**Comment by yahiheb** [comment]
@fanquake Can I know the reason why this was closed?

**Comment by jonatack** [comment]
@yahiheb I would venture to guess that it was closed because you are asking for scarce review time for bikeshedding and nitpicking about minutiae (FWIW "block chain" rather than "blockchain" is the preferred spelling in https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md). In essence, review is too scarce to be spent for this. I'd suggest looking at the issues tagged as "Good first issues".

**Pull request closed without merging**